terraform/deployments/govuk-publishing-platform/app_publisher.tf

```diff
@@ -46,21 +113,36 @@ module "publisher_public_alb" {
 # Sidekiq Worker Service
 #
 module "publisher_worker" {
-  service_name                     = "publisher-worker"
-  cluster_id                       = aws_ecs_cluster.cluster.id
+  service_name = "publisher-worker"
+  command      = ["foreman", "run", "worker"]
+  cluster_id   = aws_ecs_cluster.cluster.id
+  extra_security_groups = [
+    module.publisher_web.security_group_id,
+    local.govuk_management_access_security_group,
+    aws_security_group.mesh_ecs_service.id
+  ]
+  environment_variables            = local.publisher_defaults.environment_variables
   mesh_name                        = aws_appmesh_mesh.govuk.id
   subnets                          = local.private_subnets
   service_discovery_namespace_id   = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.id
   service_discovery_namespace_name = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.name
   source                           = "../../modules/app"
-  vpc_id                           = local.vpc_id
-  extra_security_groups            = [module.publisher_web.security_group_id, local.govuk_management_access_sg_id, aws_security_group.mesh_ecs_service.id]
-  environment_variables            = {} # TODO
-  secrets_from_arns                = {} # TODO
+  secrets_from_arns                = local.publisher_defaults.secrets_from_arns
   log_group                        = local.log_group
   aws_region                       = data.aws_region.current.name
-  cpu                              = 512
-  memory                           = 1024
+  cpu                              = local.publisher_defaults.cpu
+  memory                           = local.publisher_defaults.memory
   task_role_arn                    = aws_iam_role.task.arn
   execution_role_arn               = aws_iam_role.execution.arn
+  vpc_id                           = local.vpc_id
+}
+
+module "publisher_network_config" {
+  source  = "../../modules/task-network-config"
```

Comment:
At some point, we could probably consider moving the `task-network-config` module into the app module, since every app has some network config.

Out of scope for this PR though.

Reply:
Good idea. I've made a start by moving this block into the app module.